### ISSUE TITLE
Followup on #867 vectorize Internet Archive header logo

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#949]
@seabelis, up for another (smaller) challenge to help us vectorize the "Internet Archive" logo in our black header bar?

**Comment by seabelis** [comment]
Sure thing. But probably not until next week if that's okay.

**Comment by mekarpeles** [comment]
No deadline, just another opportunity to improve the visuals of OL :)
Thank you Lisa!

**Comment by nlevitt** [comment]
These are some comments I made to Mek yesterday
> Noah Levitt [13:51]
> suggest she start with https://archive.org/download/archipelagos/LogoAndWordmark.svg
> n.b. that's white on transparent
> which is why you probably don't see anything :)
> problem is the text is too small when you scale it down
> so i'd try scaling up only the text part if possible
> might need to move the letters closer together too

**Comment by seabelis** [comment]
Scaling up the text is going to make it super-long; even if I reduce the spacing. Any chance of putting the words on two lines or is this a violation of your branding guidelines? Are there any layout variants?

**Comment by seabelis** [comment]
Maybe it's okay. Here is a draft.
![logoandwordmark-tiny](https://user-images.githubusercontent.com/17739465/40884988-74f333ea-671e-11e8-98bf-4a6cf71f1e58.jpg)
I'm wondering if the typeface is the same as what is currently on the site. They look different, but might just be the small scale.

**Comment by LeadSongDog** [comment]
The typeface is different, but it seems looking at the main archive.org site that there is no consistency of styling there. The earliest archived pages' version used an allcaps double spaced sanserif name between twin thumbnail photos of the building: I N T E R N E T   A R C H I V E
Later versions were usually title cased and kerned, but quite often monospaced. Serifs are also inconsistent. The photos were replaced by a drawing, and then the pillar count was reduced to four in a later drawing. Today, the icon is used on IA as an alternative to the name, not paired with it. 

Usage of the combined IA logo with text on OL has been distinct from the IA site, with the text allcaps, kerned, in a seriffed font with first letters 'I' and 'A' slightly taller. The vector drawing is best seen on IA in the corner of pages streamed for a book: it is clean, minimalist, all lines straight except the classical curve of the pillars. https://archive.org/stream/princemmach00mach?ref=ol#page/31/mode/1up

My personal preference for the text part would be title case, kerned, sans serif. The clean serifs in the new OL logo are nice, but inconsistent with the rest of the site.

**Comment by mekarpeles** [comment]
@seabelis the fonts are indeed different -- the font you've used in this design is the one which is faithful to archive.org's current logo (which I think is the right way to go). The other font (currently featured on openlibrary.org for the IA logo) is Trajan/Cinzel google font, and I don't think it's what we want to use.

One thing we could try is making the font smaller than the parthenon (as is the case on archive.org and openlibrary.org's current attempt).

**Comment by nlevitt** [comment]
@seabelis commented 2 days ago
> Any chance of putting the words on two lines or is this a violation of your branding guidelines?

I think that would be fine. Correct me if I'm wrong @mekarpeles

**Comment by mekarpeles** [comment]
@nlevitt @seabelis there shouldn't be any "violation" but it may be challenging for our current design (it may force us to increase the height of our header bar which would give it more emphasis than perhaps is useful for the user.

**Comment by nlevitt** [comment]
@mekarpeles I don't think she was proposing to increase the height of the image (??)

**Comment by mekarpeles** [comment]
Worth a try!

**Comment by seabelis** [comment]
@mekarpeles Stacking the text will indeed make it taller, so I will rule that out.

I could use some clarification on what is actually needed here. The logo provided to me by @nlevitt  is already in a vector format; do you just need me to export as other vector formats (.eps, .ai)? 

@nlevitt 's comments suggested some modification was needed for the text in relation to the graphic element, so those are the changes I applied to the posted draft. (I didn't change the font itself as it is already converted to outlines, I just scrunched it together a bit.) @mekarpeles , your suggestion to keep the graphic element larger than the text is kind of the opposite of @nlevitt 's if I understand correctly.

**Comment by LeadSongDog** [comment]
The .svg format seems to work fine elsewhere, including the OL logo.  There are some other .png that also should be replaced by .svg equivalents:
https://openlibrary.org/images/logo_IA-sm.png
https://openlibrary.org/images/icons/icons_sprite.png
https://openlibrary.org/static/images/search-lens.png
https://openlibrary.org/static/images/down-arrow.png

**Comment by seabelis** [comment]
I'm still waiting for some clarification on this. I'll be doing some traveling for the next few weeks, so probably won't be able to revisit this until mid/end of July.

**Comment by mekarpeles** [comment]
@LeadSongDog, if it's okay, I'd love to keep this issue focused on the IA logo on openlibrary.org's header which currently isn't svg (and its one of the first things the user sees, impacts first impression)

re: the svg design, as has been suggested, I don't think we want 2 rows for this use case.
I think this is close: https://github.com/internetarchive/openlibrary/issues/949#issuecomment-394148094. We may want to text to be a bit smaller (similar to the proportions of the IA logo currently on openlibrary.org). The font you currently have in your svg I think is the right one to use.

Will defer to @nlevitt if he has any conflicting opinions :)

Happy travels and thanks for your efforts!

**Comment by LeadSongDog** [comment]
@mekarpeles Sure it's okay. There should likely be a separate umbrella issue to capture all these cases, with another one specific for each in turn, but I'll leave that up to you. I agree this one should be highest priority among them. I just didn't want to see the more subtle icons overlooked.

On another tangent, should the words "Internet Archive" actually be part of the header logo? It precludes the possibility of presenting it in the user's preferred language. If just the pillar-icon was used, together with a text styling, then it could just as easily read "Archivo de Internet" for a Spanish user, or "أرشيف الإنترنت" for an Arabic user.

**Comment by nlevitt** [comment]
@LeadSongDog In my humble opinion as a sort of "brand name" it should remain "Internet Archive" for all languages. Most often brand names are not translated. Maybe in the alt text we could put the translation in parentheses, something like that. But, this should be an organization-level policy and I don't know if an official choice has been made.

Nevertheless, making "Internet Archive" a text element on the page is a valid option.

**Comment by mekarpeles** [comment]
Just revisiting -- I think the current svg is pretty close. I think the text may need to be a little larger but not the same size as the parthenon

**Comment by jdlrobson** [comment]
This is what we are looking to replace (the png), correct?
![screen shot 2018-10-18 at 9 17 59 pm](https://user-images.githubusercontent.com/148752/47197464-50531500-d31b-11e8-873a-522692d0bf30.png)

**Comment by jdlrobson** [comment]
Duplicate of #783 ?

**Comment by LeadSongDog** [comment]
@jdlrobson, no, that issue is for a different image, the openlibrary icon

**Comment by LeadSongDog** [comment]
@jdlrobson  
Perhaps https://upload.wikimedia.org/wikipedia/commons/8/84/Internet_Archive_logo_and_wordmark.svg would be a useful starting point?
